# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1458322076 14400
#      Fri Mar 18 13:27:56 2016 -0400
# Node ID 2c9382700a7eec394f888f3f2d3d9de2a50e2c64
# Parent  4fc04d04dd9c3a88ee9bdba75b3f40eb89a881f1
* libinterp/template-inst/Array-sym.cc: Delete unused file.

diff --git a/libinterp/template-inst/Array-sym.cc b/libinterp/template-inst/Array-sym.cc
deleted file mode 100644
--- a/libinterp/template-inst/Array-sym.cc
+++ /dev/null
@@ -1,39 +0,0 @@
-/*
-
-Copyright (C) 1996-2015 John W. Eaton
-
-This file is part of Octave.
-
-Octave is free software; you can redistribute it and/or modify it
-under the terms of the GNU General Public License as published by the
-Free Software Foundation; either version 3 of the License, or (at your
-option) any later version.
-
-Octave is distributed in the hope that it will be useful, but WITHOUT
-ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
-FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
-for more details.
-
-You should have received a copy of the GNU General Public License
-along with Octave; see the file COPYING.  If not, see
-<http://www.gnu.org/licenses/>.
-
-*/
-
-// Instantiate Arrays of octave_child objects.
-
-#ifdef HAVE_CONFIG_H
-#  include "config.h"
-#endif
-
-#include "Array.h"
-#include "Array.cc"
-
-#include "ovl.h"
-#include "symtab.h"
-
-typedef symbol_record* symbol_record_ptr;
-
-NO_INSTANTIATE_ARRAY_SORT (symbol_record_ptr);
-
-INSTANTIATE_ARRAY (symbol_record_ptr, OCTINTERP_API);
